tutorial/about-tutorial.md

```diff
@@ -91,9 +91,9 @@ Conversational procedural structure, with a few ordered steps described in the b
 No outcome statement.
 ```
 
-You get a more conversational flow with this tutorial style, however some of the prescriptive clarity is lost by not having the headings.
+You get a more conversational flow with this tutorial style, however, some of the prescriptive clarity is lost by not having the headings.
```

Comment:
I'd actually convert the first comma to a semicolon:

```suggestion
You get a more conversational flow with this tutorial style; however, some of the prescriptive clarity is lost by not having the headings.
```

Reply:
Sure thing.